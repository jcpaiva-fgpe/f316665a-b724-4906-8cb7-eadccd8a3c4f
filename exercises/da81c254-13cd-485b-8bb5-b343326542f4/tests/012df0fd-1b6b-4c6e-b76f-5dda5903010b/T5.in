16 15
17 13
