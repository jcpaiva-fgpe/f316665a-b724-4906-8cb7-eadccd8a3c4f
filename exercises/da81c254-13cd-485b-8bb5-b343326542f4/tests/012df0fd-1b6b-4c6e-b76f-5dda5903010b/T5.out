Passaram apenas 58 minutos!
De facto!
